_content/canon/dhammapada-way-of-truth_feldmeier-peter.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Dhammapada: The Way of Truth"
+translator: feldmeier-peter
```

Comment:
These translations of whole books that are in that book's course (i.e. a Dhp translation in `course: dhp`) get put in `monographs` with the translator as `authors: []`